.github/workflows/run-tests.yml

```diff
@@ -580,6 +597,23 @@ jobs:
           path: ~/.cache/go-build
           key: go-${{ runner.os }}${{ runner.arch }}-build-${{ env.COMMIT }}
 
+      - name: Get job ID from workflow run
+        id: get_job_id
+        run: |
+          set -euo pipefail
```

Comment:
This needs to be applied to all jobs.

Reply:
ok

Reply:
I see this PR was merged without addressing this comment.